src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -32,4 +33,30 @@ default boolean invocationPresetsEnable()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "chestAudioEnable",
+		name = "Enable Chest Audio",
+		description = "Plays the audio file `toa-chest.wav` from the .runelite folder whenever a purple chest is opened",
```

Comment:
We should provide a default sound as well, then maybe this becomes an enum `NONE, DEFAULT, CUSTOM`

Reply:
Per our discord discussion I decided to not include a default sound because its 1) hard to find a good sound & 2) I suck at composing music.